### PR TITLE
Convert users_randomly_generate sproc to model function

### PR DESCRIPTION
**Pull request by jonatanschroeder**
Depends on #10384.

### CODE REVIEW

File: apps/prairielearn/package.json [hunk 1]
```diff
@@ -21,6 +21,7 @@
     "@aws-sdk/client-sqs": "^3.631.0",
     "@aws-sdk/credential-providers": "^3.631.0",
     "@aws-sdk/lib-storage": "^3.631.0",
+    "@faker-js/faker": "^8.4.1",
```
Comment (nwalters512) [inline apps/prairielearn/package.json:24]: This is a chunky old package: https://bundlephobia.com/package/@faker-js/faker@8.4.1. Per npm, it's almost 11MB unpacked.

Can we get away with a relatively small hardcoded list of first/last names?

Comment (jonatanschroeder) [reply apps/prairielearn/package.json:24]: Alternatives:
* https://www.npmjs.com/package/casual (very small, but 5 years old)
* https://www.npmjs.com/package/minifaker (2 years old)
* https://www.npmjs.com/package/@ngneat/falso

Comment (nwalters512) [reply apps/prairielearn/package.json:24]: I guess I don't see why we don't just use a hardcoded list like the sproc did? Those names are ~15kB, as opposed to at least 400kB for the smallest of those three.

Comment (jonatanschroeder) [reply apps/prairielearn/package.json]: Done in 9ab9db3f65dcac5cdf21039968940db75921c16c. I copied the names from faker.js, adding up to a 44KB library, I'm happy to trim that further if you'd like.

**Review by nwalters512 — APPROVED** [review]
It is still not clear to me that we need _quite_ this many names, but 🤷

**Pull request merged**